tests/cxx/declare_symbol.h

```diff
@@ -0,0 +1,13 @@
+//===--- declare_symbol.h - test input file for iwyu ----------------------===//
```

Comment:
There's a naming convention for test files to make it easier to correlate test `.cc` files with their headers.

There are some exceptions to this, like `direct.h` and `indirect.h`, which are 'reusable' among tests, but otherwise we try to keep headers prefixed to avoid accidental coupling between tests. There's a little blurb on this here, soon to be Markdownized:
https://code.google.com/p/include-what-you-use/wiki/InstructionsForDevelopers#Running_the_Tests

I think this should be `using_import-declare.h`.